src/components/LayerUI.tsx

```diff
@@ -173,44 +180,50 @@ const LayerUI = ({
                       />
                     </Stack.Row>
                   </Island>
-                  <LockIcon
-                    checked={appState.elementLocked}
-                    onChange={onLockToggle}
-                    title={t("toolBar.lock")}
-                  />
+                  {!showZenMode && (
```

Comment:
Why hiding the lock? Even in zen mode you may still want to switch between always using the same tool or not.

Reply:
@kbariotis I think anything except the shapes might be little distracting in the distraction-free mode so removed it in Zen mode and as @dwelle mentioned [the functionality will overall get removed](https://github.com/excalidraw/excalidraw/pull/1450) will always preserve the last shape so thought of shipping it without lock icon to see how it comes out if the Lock icon is not in Zen mode.

Do you think it will lead to any issues?